code/modules/mob/living/carbon/xenomorph/XenoProcs.dm

```diff
@@ -191,6 +191,9 @@
 	if(bruteloss == 0 && fireloss == 0)
 		return
 
+	if(src.on_fire)
```

Comment:
Doesn't this mean that this also changes
1. berserker melee won't HP vamp heal when on fire in melee combat
2. drone healer strain sacrifice wont work if the target is on fire
3. warriors wont heal on melee (which will change balance, but would technically fall within the lines of preventing healing while on fire)
4. completely wastes the heal of wardens if the target is on fire
5. vamp lurker headbites healing (affects balance, but probably within the lines of this pr)
6. steelcrest enraged healing
7. all forms of fruit, drone, and queen healing
if they're on fire at all?

I could see this being a cool fire property maybe, but a little bit of fire probably shouldnt stop 100-500 of healing HP

Reply:
Healer drones and Prae warden already can't heal a target that's on fire. they need to extinguish them first.
So they can't waste their abilities
Warrior already can't heal while on fire
All fruit and queen healing already can't be used while on fier

This will affect ravager, vamp lurker, steel crest, weed healing abilities.